test/mocha/helpers.js

```diff
@@ -41,30 +41,39 @@ function getRandomIntInclusive(min, max) {
 }
 
 // test various sequence numbers at edge case zones
-exports.sequenceNumberTests = [
-  // low values
-  ['0', 0],
-  ['1', 1],
-  // near INT32_MAX
-  ['2**31-1', 2 ** 31 - 1],
-  ['2**31', 2 ** 31],
-  ['2**31+1', 2 ** 31 + 1],
-  // near UINT32_MAX
-  ['2**32-1', 2 ** 32 - 1],
-  ['2**32', 2 ** 32],
-  ['2**32+1', 2 ** 32 + 1],
-  // in range [UINT32_MAX + 1, Number.MAX_SAFE_INTEGER]
-  ['in range [2**32, MAX_SAFE_INTEGER] (middle)',
-    2 ** 32 + (Number.MAX_SAFE_INTEGER - 2 ** 32 - 1) / 2],
-  ['in range [2**32, MAX_SAFE_INTEGER] (random)',
-    getRandomIntInclusive(2 ** 32, Number.MAX_SAFE_INTEGER)],
-  // near Number.MAX_SAFE_INTEGER
-  ['MAX_SAFE_INTEGER-1', Number.MAX_SAFE_INTEGER - 1],
-  ['MAX_SAFE_INTEGER', Number.MAX_SAFE_INTEGER],
-].map(d => ({
-  title: `should insert a document with sequence number ${d[0]}`,
-  sequence: d[1]
-}));
+
+exports.sequenceNumberTests = ({update = false}) => {
```

Comment:
This will remove the need to specify `sequenceNumberTests({})` for the common case.

```suggestion
exports.sequenceNumberTests = ({update = false} = {}) => {
```